src/services/pcn-router/datamodel/router.yang

```diff
@@ -19,35 +19,19 @@ module router {
 
   uses "polycube-standard-base:standard-base-yang-module" {
      augment ports {
-       leaf ip {
-        type inet:ipv4-address;
-        mandatory true;
-        description "IP address of the port";
-        polycube-base:cli-example "207.46.130.1";
-      }
-
-      leaf netmask {
-        type inet:ipv4-address;
-        mandatory true;
-        description "Netmask of the port";
-        polycube-base:cli-example "255.255.255.0";
+      leaf ip {
+        type inet:ipv4-prefix;
+        description "IP address and prefix of the port";
+        polycube-base:cli-example "10.0.0.1/24";
       }
 
       list secondaryip {
         description "Secondary IP address for the port";
```

Comment:
description "Secondary IP address for the port";
==>
description "Additional IP addresses for the port";